### ISSUE TITLE
Linting non-inline scripts

### ISSUE THREAD

**Issue by KTibow** [BenoitZugmeyer/eslint-plugin-html#135]
## Description

Right now, you can only lint non-inline scripts. It'd be good if you could lint stuff using the syntax of
```js
<script src="/confetti.js"></script>
```
Ideally it would use the syntax of `**/confetti.js`, because projects might have it in a different place than the root or the same dir.

## Alternatives

Use some kind of plugin to replace it with an inline script.

**Comment by brettz9** [comment]
Just to suggest another alternative. With my project [es-file-traverse](https://github.com/brettz9/es-file-traverse), you can point to an HTML file (or a file with import or require) to extract the script file names to be linted. The CLI currently has no option for base directory, but you should be able to use the programmatic API to adapt it.

**Comment by BenoitZugmeyer** [comment]
ESLint works on a per-file basis. Linting JS files referenced in HTML files is out of scope for this plugin. The alternative pointed by @brettz9 looks like a fine alternative.
